### DISCUSSION TITLE
Error in setPhaseType after updating NeqSim (Python) to latest version

### DISCUSSION THREAD

**Discussion by ngom52** [equinor/neqsim#1050] [Q&A] [answered]
Hello everyone,

I have been using the Python version of NeqSim, and recently I have updated it to the latest version. After updating, this line of my code no longer works:

gas[index].setPhaseType(0, "gas")

I received the following error:

java.lang.RuntimeException: java.lang.RuntimeException: neqsim.util.exception.InvalidInputException: PhaseType:byName - Input name is not valid.

I hope that someone can recommend to me the correct syntax. Also it would be helpful if someone can point out where I can find the changelog for different NeqSim versions

Thanks in advance!

**Comment by EvenSol** [comment, marked as answer]
Thanks for pointing out. I will look into it.
@asmfstatoil might be the best to answer this.

It seems like a solution could be to use capital letters.
gas[index].setPhaseType(0, "GAS")

**Reply by ngom52** [reply]
Hi, thanks for the solution. I have confirmed that the issue has been resolved by using capital letters

**Reply by EvenSol** [reply]
Change log is found here:
https://github.com/equinor/neqsim/releases

**Reply by ngom52** [reply]
Thanks!
